src/tests/HDFFile.cpp

```diff
@@ -24,7 +24,11 @@ using std::chrono::milliseconds;
 using std::chrono::steady_clock;
 using nlohmann::json;
 
-MainOpt getTestOptions() { return MainOpt(); }
+MainOpt getTestOptions() {
+  MainOpt mo;
```

Comment:
Please use LLVM naming: upper CamelCase for variables, perhaps `TestOptions` or something like that.